packages/core/src/browser/quick-open/quick-command-service.ts

```diff
@@ -191,7 +191,10 @@ export class QuickCommandService implements QuickOpenModel, QuickOpenHandler {
         raw.forEach((command: Command) => {
             if (command.label) {
                 const contexts = this.contexts.get(command.id);
-                if (!contexts || contexts.some(when => this.contextKeyService.match(when))) {
+                if (!contexts || contexts.some(when => {
+                    when = when?.trim();
+                    return when === undefined || when === 'true' || this.contextKeyService.match(when);
+                })) {
```

Comment:
`this.contexts` is defined as:

```ts
protected readonly contexts = new Map<string, string[]>();
```

I do not think in this area that when can be `undefined`:

```suggestion
                if (!contexts || contexts.some(when => {
                    when = when.trim();
                    return when === 'true' || this.contextKeyService.match(when);
                })) {
```

Reply:
Done, Thanks Vince !  A new commit is now available with your proposed change.